### PR TITLE
Update plug_rest.gen.resource to add path option

### PR DESCRIPTION
**Pull request by yuchunc**

### CODE REVIEW

File: lib/mix/tasks/plug_rest.gen.resource.ex [hunk 1]
```diff
@@ -64,8 +64,15 @@ defmodule Mix.Tasks.PlugRest.Gen.Resource do
...
-    create_directory Path.dirname(file)
+    file =
+      case String.slice(opts[:path], -3..-1) == ".ex" do
+        true ->
+          opts[:path]
+        false ->
+          Path.join([opts[:dir], underscored]) <> ".ex"
+      end
+
+    file |> IO.inspect |> Path.dirname |> create_directory
```
Comment (christopheradams) [inline lib/mix/tasks/plug_rest.gen.resource.ex]: remove the left-over `IO.inspect`

**Comment by christopheradams** [comment]
I think at this point we can drop the `dir` option and just support `path`.

If the `path` doesn't include the filename, then it can behave like `dir` did before.

What do you think?

File: lib/mix/tasks/plug_rest.gen.resource.ex (hunk 1 above)
Comment (yuchunc) [reply lib/mix/tasks/plug_rest.gen.resource.ex]: oops!

**Comment by yuchunc** [comment]
I think remove or not are both justify. Don't remove, less surprise; remove, cleaner code.
I don't really have a preference, let me know which way you prefer, I'll update the code.

**Pull request merged**